BIPs/2023-W22/BIP-316a.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "1.0",
+  "chainId": "1",
+  "createdAt": 1685637015445,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.14.1",
+    "createdFromSafeAddress": "0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f",
+    "createdFromOwnerAddress": "",
+    "checksum": "0x42a37d051065adb6e4cec16efe5aed4ec97a9a5e36eb1624ec330941b3595bbf"
+  },
+  "transactions": [
```

Comment:
The DAO Multisig eth:0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f will interact with the AuthorizerAdapterEntryPoint 0xf5dECDB1f3d1ee384908Fbe16D2F0348AE43a9eA with the following arguments:

target: 0xE867AD0a48e8f815DC0cda2CDb275e0F163A480b which corresponds to the veBAL gauge
data: 0xab8f0945 which corresponds to the function for kill()